mysql-test/suite/tianmu/t/issue781.test

```diff
@@ -0,0 +1,76 @@
+--source include/have_tianmu.inc
+
+USE test;
+
+--disable_warnings
+
+## DDL
+
+DROP TABLE IF EXISTS cs1;
+
+CREATE TABLE cs1 (d1 int, d2 int) ENGINE=TIANMU;
```

Comment:
how about the filter is on PK. For example:

create table tb1 (d1 int primary key, d2 int) engine =tianmu;

select xxxx from tb1 where d1 in (xxx);